### ISSUE TITLE
RiskRelease out of order execution

### ISSUE THREAD

**Issue by a1eksandre** [mzheravin/exchange-core#83]
Based on the disruptor configuration in [ExchangeCore](https://github.com/mzheravin/exchange-core/blob/2a9de802d1c5f0605e7bae494d529031c7d3e2af/src/main/java/exchange/core2/core/ExchangeCore.java#L159), order execution should take following steps: 

1. `preProcessCommand` of RiskEngine
2. `processOrder` of MatchingEngineRouter 
3. `handlerRiskRelease` of RiskEngine

As I observed during testing, when order place event is processed, `handlerRiskRelease` runs before `processOrder`. 
If a matching order was found and order was closed, corresponding users' accounts are not updated with the trade results, until the next event is processed. 
As the next event runs `handlerRiskRelease` and `preProcessCommand` before `processOrder`, users' accounts are in a correct state during order processing, but I believe described flow is not intentional.
In short, instead of processing event in the order described in the beginning, actual flow is:
1. `handlerRiskRelease`  2. `preProcessCommand`  3. `processOrder`
